### PR TITLE
Updating comments to use proto var names - toward #1252

### PR DESCRIPTION
**Pull request by jmank88**
Went through the proto files in the roachpb package.

### CODE REVIEW

File: roachpb/api.proto [hunk 1]
```diff
@@ -129,7 +129,7 @@ message ConditionalPutRequest {
   optional Span header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
   // The value to put.
   optional Value value = 2 [(gogoproto.nullable) = false];
-  // ExpValue.Bytes empty to test for non-existence. Specify as nil
+  // exp_value.bytes empty to test for non-existence. Specify as nil
```
Comment (tschottdorf) [inline roachpb/api.proto]: change that to a complete sentence while you're at it.

File: roachpb/api.proto [hunk 2]
```diff
@@ -252,7 +252,7 @@ message EndTransactionResponse {
 }
 
 // An AdminSplitRequest is the argument to the AdminSplit() method. The
-// existing range which contains Span.Key is split by
+// existing range which contains header.key is split by
 // split_key. If split_key is not specified, then this method will
 // determine a split key that is roughly halfway through the
 // range. The existing range is resized to cover only its start key to
```
Comment (tschottdorf) [inline roachpb/api.proto]: l. 290 mentions `subsumed_range` which should just be `subsumed range`.

**Comment by tschottdorf** [comment]
LGTM after small comments.

**Comment by jmank88** [comment]
No problem. Good way to get to know the code base, and satisfy my tendency for hyper organization ;)

File: roachpb/api.proto (hunk 1 above)
Comment (jmank88) [inline roachpb/api.proto]: Fixed. Found a few I missed above here as well.

**Comment by tschottdorf** [comment]
LGTM. If you're looking for another (admittedly, small) cleanup, check out https://github.com/cockroachdb/cockroach/issues/2332. Thanks again.

**Pull request merged**